src/Business/Detection/Repositories/IDetection.cs

```diff
@@ -0,0 +1,10 @@
+namespace Business.Detection.Repositories
+{
+    using System.Collections.Generic;
+    using Business.Detection.Models;
+
+    public interface IDetection
```

Comment:
Would the name: `IDetectionRepository` be more relevant here?

Reply:
Done!